test/controllers/api/deploys_controller_test.rb

```diff
@@ -3,17 +3,17 @@
 SingleCov.covered!
 
 describe Api::DeploysController do
+  assert_route "/api/deploys/active_count", 'api/deploys', 'active_count'
```

Comment:
yes! one more nitpick though, I would nest it inside 

```ruby
describe 'routing' do
  it 'dispatches correctly' do
    ...
  end
end
```

not sure what minitest's behaviour is if a spec fails outside of an `it` block, maybe it's fine.